packages/frontend-vitals-monitoring/rollup.config.js

```diff
@@ -0,0 +1,17 @@
+import typescript from "rollup-plugin-typescript2";
+
+export default {
+  external: ["forwarded-parse", "pino"],
```

Comment:
`forwarded-parse` can probably be deleted here I think?

Reply:
Confirmed as resolved